extensions/sql-database-projects/src/sqldbproj.d.ts

```diff
@@ -73,6 +73,18 @@ declare module 'sqldbproj' {
 		 */
 		addSqlCmdVariable(name: string, defaultValue: string): Promise<void>;
 
+		/**
+		 * Adds DatabaseSource property to the project
+		 * @param databaseSource Source of the database to add
+		 */
+		addDatabaseSourceToProjFile(databaseSource: string): Promise<void>;
+
+		/**
+		 * Removes DatabaseSource property from the project
+		 * @param databaseSource Source of the database to remove
+		 */
+		removeDatabaseSourceFromProjFile(databaseSource: string): Promise<void>;
```

Comment:
what's the scenario where the DatabaseSource will be removed? Would it make more sense for `createProject()` to accept an optional databaseSource parameter instead of exposing these two apis?

Reply:
Hm... I'd assume if something can be added - there should be an ability to remove it as well. If the question was "Do you plan to call it from your extension right now?", then the answer is likely no.

Reply:
In conclusion, is it reasonable to keep this `removeDatabaseSource` function, despite it not being called?

Reply:
sure we can keep it in case it's needed in the future